docs/tutorials/dotnet-systemd.md

```diff
@@ -178,19 +194,19 @@ Return to your Windows host and reconnect to your Bot Emulator using the same in
 
 You can stop your bot from running at any time with the command:
 
->`$ sudo systemctl stop echoes.service`
+> `sudo systemctl stop echoes.service`
 
 ## Tutorial complete!
 
 You now have a simple Echo Bot running as a systemd service on WSL that you can access from your host Windows machine.
 
 If you would like to expand on this example try reviewing some of the more advanced [Bot Framework samples](https://github.com/Microsoft/BotBuilder-Samples/blob/main/README.md) on the Microsoft GitHub.
 
-To read more about how Ubuntu supports .Net developers, making it easier than ever to build multi-platform services and applications, read our [recent announcement](https://ubuntu.com/blog/install-dotnet-on-ubuntu).
+To read more about how Ubuntu supports .NET developers, making it easier than ever to build multi-platform services and applications, read our [recent announcement](https://ubuntu.com/blog/install-dotnet-on-ubuntu).
```

Comment:
Aug. 2022 is not recent anymore.

Reply:
True indeed! Thanks for all your comments.